packages/fluentui/react-northstar/src/components/Flex/FlexItem.tsx

```diff
@@ -58,7 +58,7 @@ const applyStyles = (
   // if element is DOM element
   if (typeof element.type === 'string') {
     return React.cloneElement(element, {
-      className: cx(element.props.deprecated_className, classes.root),
+      className: cx(element.props.className, classes.root),
```

Comment:
Please how would that work with class components, which still have `deprecated_className` ?

Reply:
This was unintended change where we are using the prop className, not the static field, but I accidentally replaced it with `deprecated_className` in one of the prev PRs.